gettsim/parameters/eink_st_abzuege.yaml

```diff
@@ -406,6 +410,14 @@ alleinerziehenden_freibetrag:
   2015-01-01:
     scalar: 1908
     reference: Art. 1 G. v. 16.07.2015 BGBl. I S.1202
+  2020-01-01:
+    scalar: 4008
+    reference: Art. 1 Nr. 5 G. v. 29.06.2020 BGBl. I S.1512
+    note: Temporäre Corona-Hilfe für 2020 und 2021
+  2022-01-01:
```

Comment:
Parameter seems to be changed to be permanent 4008 https://www.bundestag.de/dokumente/textarchiv/2020/kw51-de-jahressteuergesetz-2020-812872, https://www.buzer.de/3_JStG_2020.htm.

We also don't have the additional 240 Euro:
> Für jedes weitere Kind im Sinne des Absatzes 1 erhöht sich der Betrag nach Satz 1 um 240 Euro je weiterem Kind.

Reply:
good catch. is changed. We have the €240 though:

https://github.com/iza-institute-of-labor-economics/gettsim/blob/079608c8ac3fd3bfe032f5d39ebb8bd8a3b3be55/gettsim/parameters/eink_st_abzuege.yaml#L410-L421